src/VectorBasemapLayer.js

```diff
@@ -138,12 +138,16 @@ export var VectorBasemapLayer = VectorTileLayer.extend({
     map.removeLayer(this._maplibreGL);
 
     if (map.attributionControl) {
+      Util.removeEsriAttribution(map);
```

Comment:
If using with the previous version of Esri Leaflet (Esri Leaflet v3.0.11), this line errors. I think we may want to wrap this in a check to see if the function exists.